src/containers/App.js

```diff
@@ -130,13 +131,18 @@ const App = React.createClass({
             <strong>{contrast.score(colorOne, colorTwo)}</strong> / <strong>{contrast.ratio(colorOne, colorTwo).toFixed(2)}</strong>
           </p>
         </div>
-        <Combination colorOne={colorOne} colorTwo={colorTwo} />
+        <ReactCSSTransitionGroup
+          transitionName="color-switch"
+          transitionEnterTimeout={1300}
```

Comment:
Could we lower these durations? I feel like I'm waiting for a huge process to occur while the fading happens :neutral_face: 

256, perhaps?